### PR TITLE
ZOOorder FAQ and Layout setup

### PR DESCRIPTION
**Pull request by dguzenko**

### CODE REVIEW

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    category: docs
+---
+
+* **Where can the default ordering of items is set?** - It is configured in the application settings.
+
+* **Is it possible to set ordering of items within one category?** - ZOOorder allows to add the module to the category page where you can specify the ordering params for this category.
+
+* **Why the ordering of numbers is not correct? (E.g. 700 is higher than 1000)** - Please make sure that the Element has the correct Value Type (e.g. Numerical). You can check it in the Layout settings.
+ 
+* **Is it possible to setup ordering by several elements?** - No, it isn't, because ZOO has this limitation.
```
Comment (miljan-aleksic) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Remove the reason. Just say it's curretnly not possible.

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Where can the default ordering of items is set?** - It is configured in the application settings.
```
Comment (miljan-aleksic) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Review this question, seems there is some mistake.

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md (hunk 1 above)
Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Done

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md (hunk 2 above)
Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Fixed

File: 02.extensions/zooorder/01.basics/03.integration/docs.md [hunk 3]
```diff
@@ -12,3 +12,11 @@ Once **ZOOorder** is installed it will become available as a Joomla! module read
...
 | **Module Layout** | The layout that will be used to display the module. |
 | **Module Class Suffix** | A module class suffix for specific module styling. |
+
+## Layout Setup
+
+Configure the ZOOorder layout by assigning elements to rendering positions. 
+
+>>> More information about Element assigning can be found at [ZOO documentation](http://yootheme.com/zoo/documentation/advanced/assign-elements-to-layout-positions).
+
+For correct ordering select the corrsponding **Value Type** in the ZOOorder layout Basic settings. Among the options are *Alphabetical*, *Numerical*, *Numerical with decimals* and *By Date*.
```
Comment (ameli90) [inline 02.extensions/zooorder/01.basics/03.integration/docs.md]: plz make it "corresponding"

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Where the default ordering of items is set?** - It is configured in the application settings.
```
Comment (ameli90) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: also we can specify that items ordering on "category view"

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+* **Is it possible to set ordering of items within one category?** - ZOOorder allows to add the module to the category page where you can specify the ordering params for this category.
+
+* **Why the ordering of numbers is not correct? (E.g. 700 is higher than 1000)** - Please make sure that the Element has the correct Value Type (e.g. Numerical). You can check it in the Layout settings.
```
Comment (ameli90) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: in the ZOOorder layout settings

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ 
+* **Is it possible to setup ordering by several elements?** - No, it's currently not supported.
```
Comment (ameli90) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: maybe add "at one time"

File: 02.extensions/zooorder/01.basics/03.integration/docs.md (hunk 3 above)
Comment (dguzenko) [inline 02.extensions/zooorder/01.basics/03.integration/docs.md]: Fixed

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md (hunk 4 above)
Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Done

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md (hunk 5 above)
Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Done

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md (hunk 6 above)
Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Done

File: 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Where the default ordering of items on the Category view is set?** - It is configured in the application settings.
```
Comment (miljan-aleksic) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Change to "Where can be set the default Items ordering for the Category view?"

Comment (dguzenko) [inline 02.extensions/zooorder/03.troubleshooting/02.faq/docs.md]: Done

**Pull request merged**